metrics/density/README.md

```diff
@@ -7,7 +7,10 @@ This number of containers could be working under different configurations/condit
 - `docker_memory_usage.sh`: measures the Proportional Set Size (PSS) memory average
    for a number of containers launched in idle mode. This test uses the `sleep` command
    to allow any memory optimizations to 'settle' (e.g. KSM execution) for a configurable
-   period of time.
+   period of time. It also has an optional 'auto' mode which will try to determine when KSM
```

Comment:
Lines 10-11 suggested rewrite: This test has an optional 'auto' mode that tries to determine when KSM settles down by examining the KSM `full_scans` and `pages_shared` values.